.drone.yml

```diff
@@ -24,7 +24,7 @@ steps:
     checksum:
     - sha256
     checksum_file: CHECKSUMsum-amd64.txt
-    checksum_flatten: true
+    checksum_flattzen: true
```

Comment:
why did we modify this?

Reply:
this was modified when I ran go generate, but I will double check this again